spec/report_spec.rb

```diff
@@ -87,35 +43,7 @@
   end
 
   describe '.retrieve' do
-    it 'retrieves a payment_log report' do
-      report = described_class.create(
-        start_date: Fixture.report_start_date,
-        end_date: Fixture.report_end_date,
-        type: 'payment_log',
-      )
-
-      retrieved_report = described_class.retrieve(report.id)
-
-      expect(retrieved_report).to be_an_instance_of(described_class)
-      expect(retrieved_report.start_date).to eq(report.start_date)
-      expect(retrieved_report.end_date).to eq(report.end_date)
-    end
-
-    it 'retrieves a refund report' do
-      report = described_class.create(
-        start_date: Fixture.report_start_date,
-        end_date: Fixture.report_end_date,
-        type: 'refund',
-      )
```

Comment:
we are not testing these report types? Same for above tracker and shipment invoice

Reply:
Oops, never mind. I just saw the internal ticket requirement.